llarp/service/intro.hpp

```diff
@@ -55,8 +55,8 @@ namespace llarp
       bool
       operator<(const Introduction& other) const
       {
-        return expiresAt < other.expiresAt || pathID < other.pathID || router < other.router
-            || version < other.version || latency < other.latency;
+        return std::tie(expiresAt, pathID, router, version, latency)
+            < std::tie(other.expiresAt, other.pathID, other.router, other.version, other.latency);
```

Comment:
I'm not certain that this is the right priority for all the fields here.

Reply:
I don't think the priority here matters; the only place that is currently using this is path/pathset.cpp, and it appears to only be using it for de-duplication, not (specifically) for ordering.